k8s/client-deployment.yaml

```
@@ -0,0 +1,166 @@
+            memory: "256Mi"
+            cpu: "200m"
+        
+        # Security Context
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          runAsUser: 101  # nginx user
+          capabilities:
+            drop:
+              - ALL
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)